cypress/integration/Util.ts

```diff
@@ -0,0 +1,55 @@
+export function createInstance(newInstanceName: string) {
+  cy.ouiaId("create-smart-event-instance", "PF4/Button").click();
+  cy.ouiaId("create-instance", "PF4/ModalContent").then(($modal) => {
+    cy.wrap($modal)
+      .should("be.visible")
+      .within(() => {
+        cy.ouiaId("new-name", "PF4/TextInput").type(newInstanceName);
+        cy.ouiaId("info-instance-available-soon", "PF4/Alert").should(
+          "have.text",
+          "Info alert:Your Smart Events instance will be ready for use shortly after creation."
+        );
+        cy.ouiaId("submit", "PF4/Button").click();
+      });
+    cy.wrap($modal, { timeout: 20000 }).should("not.exist");
+  });
+}
+
+export function newInstanceStatus(instanceName: string) {
+  cy.ouiaId("Instances list table", "PF4/Table")
+    .ouiaId(instanceName, "PF4/TableRow")
+    .within(() => {
+      cy.get("td:nth-child(2)", { timeout: 30000 }).should(
+        "have.text",
+        "ready"
+      );
+    });
+}
+
+export function deleteInstance(deleteInstanceName: string) {
+  cy.ouiaId("Instances list table", "PF4/Table")
+    .ouiaId(deleteInstanceName, "PF4/TableRow")
+    .find("td")
+    .then(($cells) => {
+      expect($cells).have.length(4);
+      expect($cells.eq(1)).have.text("ready");
```

Comment:
@ChughShilpa REST api says that delete is possible only in "ready" or "failed" state. We should allow to define "failed" state here.

Reply:
Added "failed" state